node/setup.go

```diff
@@ -309,7 +300,7 @@ func createMempoolAndMempoolReactor(
 		reactor := mempl.NewReactor(
 			config.Mempool,
 			mp,
-			waitSync,
+			true,
```

Comment:
I assume this is always true because block sync is mandatory?

Reply:
Actually it was so mindless that it was wrong 🤦.
Lucky, a couple of unit tests failed, which allowed me to spot the problem and validate the fix.